proctor-consumer/src/test/java/com/indeed/proctor/consumer/ProctorGroupsWriterTest.java

```diff
@@ -44,6 +45,7 @@ public class ProctorGroupsWriterTest {
                     .put(EMPTY_ALLOCID_DEFINITION_TEST_NAME, CONTROL_BUCKET)
                     .put(GROUP1_TEST_NAME, GROUP1_BUCKET)
                     .put(SILENT_TEST_NAME, GROUP1_BUCKET)
+                    .put(FORCED_TEST_NAME, GROUP1_BUCKET)
```

Comment:
Proctor determineGroups for forced groups will add the TestBucket to the bucket map and the test definition to the Test Definition map, but will not add an allocation to the allocation map within the Proctor Result. Followed this for testing manually here